www/js/photos.js

```diff
@@ -0,0 +1,196 @@
+let dir = "photos/";
+let fileExtension = ".jpg";
+let pageSize = 9;
+let pageNumber = 1;
+let numPages = 0;
+let pageWindow = 5;
+
+let loadFunction = function () {
+    $("#photos").html('');
+    let url = dir + "?page=" + pageNumber + "&size=" + pageSize;
+
+    function activateNextAndPrev() {
+        if (1 === pageNumber) {
+            console.log('Disabling prev and enabling next')
+            $('.move-backwards').addClass('disabled');
+            $('.move-forwards').removeClass('disabled');
+        } else if (numPages === pageNumber) {
+            console.log('Disabling next and enabling prev')
+            $('.move-backwards').removeClass('disabled');
+            $('.move-forwards').addClass('disabled');
+        } else {
+            console.log('Enabling next and prev')
+            $('.move-backwards').removeClass('disabled');
+            $('.move-forwards').removeClass('disabled');
+        }
+    }
+
+    function buildPaginationLinks(response) {
```

Comment:
I'm not sure why but if there aren't many photos (or no photos) in the gallery, negative pagination links are created. Screenshot attached. 
![pagination](https://user-images.githubusercontent.com/1679375/51745917-5126e000-209c-11e9-93e5-32ad76aab7bb.jpeg)

Reply:
Whoops! The thing I didn't want to test as I have ~3000 images on my Pi that need transferring.

I'll have a fix soon enough.